src/main/java/org/zalando/nakadi/service/timeline/TimelineService.java

```diff
@@ -161,6 +163,11 @@ public TopicRepository getTopicRepository(final Timeline timeline) {
         return topicRepositoryHolder.getTopicRepository(timeline.getStorage());
     }
 
+    public TopicRepository getTopicRepositoryForTimeline(final Timeline timeline)
```

Comment:
I was wondering how I could have missed this, but git blame tells me that the identical function a few lines above appeared when I merged master into this branch, after having written this function. I'll fix this.